samples/3_map_state_simple/e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
+
+describe('simple-map App', () => {
```

Comment:
Just curious, any particular reason you kept the e2e tests but not the component unit tests?